src/scala/com/github/johnynek/bazel_deps/Writer.scala

```diff
@@ -376,7 +376,8 @@ object Writer {
       s"""# duplicates in ${coord.unversioned.asString} $status\n""" +
         vs.filterNot(e => replaced(e.source))
           .map { e =>
-            s"""# - ${e.source.asString} wanted version ${e.destination.version.asString}\n"""
+            val evicted = if (e.label) " (evicted)" else ""
```

Comment:
rather than using Boolean can we add to `DepsModel.scala` something like:

```scala
sealed abstract class EvictionState(val isEvicted: Boolean)
object EvictionState {
  case object Evicted extends EvictionState(true)
  case object Retained extends EvictionState(false)
}
```
so we don't have "boolean blindess" type bugs? I'd rather have `Edge[MavenCoordinate, EvictionState]]` and `e.label.isEvicted`